docs/deploy-hyperlane.mdx

```diff
@@ -169,7 +65,15 @@ If this is your first time using Kurtosis Cloud, you'll need to sign in with Goo
 
 ![Get excited](../static/img/kurtosis-get-excited.png)
 
-A prefilled configuration screen will pop up with all your relayer and validator information from Steps 1 and 2. The last step will be to input your `Validator Key` from Step 2. The screenshot below shows an example prefilled configuration.
+A prefilled configuration screen will pop up with all your relayer and validator information from Steps 1 and 2.
+
+:::note
+
+If you are having issues deploying agents on Kurtosis Cloud through the CLI, you can manually enter this information by clicking `Run` on the [Hyperlane Package](https://cloud.kurtosis.com/gateway/ips/18-206-3-164/ports/9711/catalog/github.com%2Fkurtosis-tech%2Fhyperlane-package).
```

Comment:
ahhh I assumed it would work if you're logged in, it's a link to the kurtosis catalogue but it seems they hardcode your account's project details into the url for some reason?